Fix OpInfo failing without scipy

heitorschueroff opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#54735 Fix OpInfo failing without scipy**

One of the tests didn't wrap scipy call with TEST_SCIPY. Also, the wrapper function seems unnecessary and requires lambdas to be created.

Differential Revision: [D27351349](https://our.internmc.facebook.com/intern/diff/D27351349)

heitorschueroff committed:
Fix OpInfo failing without scipy

[ghstack-poisoned]

mruberry reviewed (approved):
Nice!

mruberry commented:
This is great but I think you want to preserve the NOTHING or update this filer:

https://github.com/pytorch/pytorch/blob/dfc7fa03e5d33f909b9d7853dd001086f5d782a0/test/test_unary_ufuncs.py#L31

heitorschueroff committed:
Update on "Fix OpInfo failing without scipy"


One of the tests didn't wrap scipy call with TEST_SCIPY. Also, the wrapper function seems unnecessary and requires lambdas to be created.

[ghstack-poisoned]

kshitij12345 reviewed (approved):
Simple and straightforward! Great!
Thanks
